build.proj

```diff
@@ -77,6 +78,12 @@
   <UsingTask TaskName="ValidateStrongNameSignatureTask" AssemblyFile="$(LibraryToolsFolder)\Microsoft.WindowsAzure.Build.Tasks.dll"  />
   <UsingTask TaskName="FilterOutAutoRestLibraries" AssemblyFile="$(LibraryToolsFolder)\Microsoft.WindowsAzure.Build.Tasks.dll"  />
   
+  <!-- Building ClientRuntime projects -->
+  <Target Name="BuildClientRuntime">
+      <Exec Command="dotnet restore" WorkingDirectory="%(ClientRuntimeProjects.RootDir)\%(ClientRuntimeProjects.Directory)"/>
+      <Exec Command="dotnet build --configuration $(Configuration)" WorkingDirectory="%(ClientRuntimeProjects.RootDir)\%(ClientRuntimeProjects.Directory)"/>
```

Comment:
Nit: Any reason MSBuild task is not used to build, rather using Exec Command task?
Any particular reason dotnet restore is required? because MsBuild will call ResolveNugetPackageAssets